keyboards/handwired/neodox/info.json

```
@@ -0,0 +1,105 @@
+{
+    "manufacturer": "MatteoDM",
+    "keyboard_name": "Neodox",
+    "maintainer": "MatteoDM",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP8"]
+    },
+    "processor": "RP2040",
+    "url": "https://gitlab.com/neodox/neodox",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x5269",
+        "suspend_wakeup_delay": 200,
+        "vid": "0x4D69"
+    },
+    "split": {
+        "enabled": true
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "backlight": true,
+        "wpm": true,
+        "quantum_painter": true
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "label": "_Nav || `", "matrix": [0, 0], "w": 1.25, "x": 0, "y": 0.375 },
+                { "label": "1", "matrix": [0, 1], "w": 1, "x": 1.25, "y": 0.375 },
```

```suggestion
                { "label": "1", "matrix": [0, 1], "x": 1.25, "y": 0.375 },
```
etc...

`w: 1` is the default.

---

done!